VirtualList/VirtualList.js

```diff
@@ -832,6 +832,14 @@ VirtualGridList.propTypes = /** @lends sandstone/VirtualList.VirtualGridList.pro
 	 */
 	scrollMode: PropTypes.string,
 
+	/**
+	 * When it's true, the item snaps to center.
```

Comment:
```suggestion
	 * If true, the item snaps to center.
```
I referenced how it was written elsewhere.

Reply:
🤔 let's just keep it.